wallet-new/src/Cardano/Wallet/API/V1/Errors.hs

```diff
@@ -68,9 +69,14 @@ data WalletError =
     | WalletAlreadyExists
     | AddressNotFound
     | TxFailedToStabilize
+    | InvalidPublicKey { weProblem :: !Text }
```

Comment:
Field accessors in a sum types are almost always partial and hence a really bad idea. The core team has already expended a huge amount of effort removing them from the code base. Please do not add any new ones.

When we switch to ghc 8.4 [-Wpartial-fields](https://downloads.haskell.org/~ghc/master/users-guide/using-warnings.html#ghc-flag--Wpartial-fields) will be added to the default flags in the `stack.yaml` file.